ntbs-service/Services/ReportingLinksService.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.Extensions.Configuration;
+
+namespace ntbs_service.Services
+{
+    public interface IReportingLinksService
+    {
+        string GetReportingPageUrl();
+        string GetClusterReport(string clusterId);
+    }
+
+    public class ReportingLinksService : IReportingLinksService
+    {
+        private readonly IConfiguration _configuration;
+
+        public ReportingLinksService(IConfiguration configuration)
+        {
+            _configuration = configuration;
+        }
+
+        public string GetReportingPageUrl() =>
+            _configuration.GetValue<string>(Constants.ExternalLinksReportingUri);
+
+        public string GetClusterReport(string clusterId)
+        {
+            return "";
```

Comment:
Is there a ticket to complete this? If so, it would be good to put a TODO in here against that ticket

Reply:
Yep, I've added that now